### DISCUSSION TITLE
Comparing built-in C64 vs MiSTer-version: Wormhole

### DISCUSSION THREAD

**Discussion by BrettHallen** [MJoergen/C64MEGA65#8] [General]
I made two quick videos comparing the PAL disk version of "Wormhole" from Protovision.

[First video](https://youtu.be/QI3x3SHamSA) is using the M65's "GO64" command and then loading the game from a D81 disk image (created in DirMaster, copying the files from the original D64 image).

[Second video](https://youtu.be/AJ7rAJ3uXm0) is using the RC3 version of the MiSTer core and the original D64 disk image.  Very good!

Wormhole from Protovision:
https://www.protovision.games/games/wormhole.php
https://protovision.itch.io/wormhole

Brett.

**Comment by sy2002** [comment]
Thank you, @0ddjob - I am closing this post as it is now a few years old.
